### DISCUSSION TITLE
Highlight hyperlinks on https://c128lib.github.io

### DISCUSSION THREAD

**Discussion by johnpalermo** [c128lib/.github#9] [Ideas]
Hi Team,

Just an idea for the documentation page (https://c128lib.github.io).  At the moment, hyperlinks are not highlighted, so it's difficult to see what is a hyperlink and what is regular text.  What do you think about changing the theme to have hyperlinks emphasized?

**Comment by intoinside** [comment]
Yes it's a good idea, seems to be a fast css check.
The same css is used on Reference (and suffer of the same problem) so this edit can be done also there.

**Reply by intoinside** [reply]
I think it's also better to make main container responsive to browser width (this is my 1920 pixel wide screen)...
![image](https://user-images.githubusercontent.com/8144124/223112348-e2e06704-2043-490a-96ed-831a210ece8d.png)

**Reply by intoinside** [reply]
Fixed (on both)
